client/src/CasesImport/CasesImport.js

```diff
@@ -14,48 +13,62 @@ export function CasesImport() {
 
   const columns = getColumns(t)
 
+  const getType = file => {
+    const nameArray = file.name.split('.')
+    const extension = nameArray[nameArray.length - 1]
+    // sometimes xls files come in with an empty type so we want to parse
+    // the extension to verify what parser to use
+    if (file.type === '' && (extension === 'xls' || 'xlsx')) {
+      return 'application/vnd.ms-excel'
+    } else {
+      return file.type
+    }
+  }
+
   const onFileChange = e => {
     setError('')
 
     const [file] = e.target.files
-    const { type } = file
-    if (!(type in parserTypes)) {
-      setKids([])
-      setError(t('uploadCasesInvalidFileFormat'))
-      return
-    }
+    if (file) {
```

Comment:
A minor thing. AFAIK, `file` will never we falsy since `onFileChange` is only called when a file is uploaded.

Reply:
Will fix

Reply:
I actually think it is empty if you cancel out of the file menu; just remembered I did that on purpose last night.

Reply:
Ah, interesting.

Reply:
I just tried to see if the `onChange` event is triggered when the file explorer is closed without selecting a file but it wasn't. Not sure if I am missing something 😕